unlock-app/src/components/interface/buttons/overlay/ConfirmedKey.js

```diff
@@ -15,20 +16,27 @@ ConfirmedKey.propTypes = {
   hideModal: PropTypes.func.isRequired,
 }
 
-const NotHover = styled(Svg.Checkmark)``
+export const NotHover = styled(Svg.Checkmark)``
```

Comment:
I think you are not actually using these? In that case, maybe the export statement is superfluous?

Reply:
Oops, accidentally left that in from when I tried a few different ways to get the hover. Good catch.